README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+**Docker Swarm standalone**: This project. An API proxy system in which a single Docker host fronts several Docker Engines to make them appear as if they are a single Docker Engine.
```

Comment:
Sorry I didn't realize this earlier, but the description at the top of this file and this one are differently worded. Additionally, Swarm is much more than an `API proxy system`, so we should change the language here.